pubtools/_pulp/tasks/garbage_collect.py

```diff
@@ -62,6 +69,22 @@ def run(self):
 
         LOG.info("Temporary repo(s) deletion completed")
 
+        # Clear out old all-rpm-content
```

Comment:
May I suggest putting this into a separate method rather than making run() bigger?

In fact the best would be also to refactor the existing code to end up with something like this:

```python
def run(self):
    self.delete_temp_repos()
    self.clean_all_rpm_content()

@step("Delete temporary repos")
def delete_temp_repos(self):
    ... # all the logic previously in run()

@step("Clean all-rpm-content")
def clean_all_rpm_content(self):
    ... # the new logic you're adding
```

The `@step` decorator puts some standard logging around sub-steps of a task.